### DISCUSSION TITLE
How to maximize window

### DISCUSSION THREAD

**Discussion by EtherGhost** [FreeTubeApp/FreeTube#1948] [Q&A]
How do you maximize the window? v0.15.1 Beta
![Skärmbild från 2021-12-18 22-21-29](https://user-images.githubusercontent.com/40342638/146655794-6e15f589-4357-4768-85a1-ff734f4acc48.png)

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
what OS and Desktop Environment are u using?

**Reply by EtherGhost** [reply]
Manjaro gnome. I noticed that I can edit a text file to set it to 1920x1080. I'm on a phone now so I can't check what file that was.

**Reply by efb4f5ff-1298-471a-8973-3d47447115dc** [reply]
If ur using Manjaro that means that u have installed FreeTube from the AUR right?

I'm assuming this because Manjaro is based of Arch

**Reply by EtherGhost** [reply]
If I remember it correctly yes

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
From our readme.md

> These builds are maintained by the community. While they should be safe, download at your own risk. There may be issues with using these versus the official builds. Any issues specific with these builds should be sent to their respective maintainer.

I would advise u to reach out to the maintainer of the package u installed.

**Reply by EtherGhost** [reply]
Are there suppose to be a maximize button?

**Reply by efb4f5ff-1298-471a-8973-3d47447115dc** [reply]
yes. installed Freetube in a Manjaro Gnome vm and i see a maximize button. 
![Capture](https://user-images.githubusercontent.com/73130443/148680758-4c4a828b-330a-484b-afd0-06978d99e837.PNG)

used this guide for the commands 
https://wiki.manjaro.org/index.php/Arch_User_Repository

`pamac install base-devel git`
`pamac build freetube-bin`

If u want to go this route i would advise u to create a backup of ur history and subscriptions.

Settings > Data Settings > Export Subscriptions / Export History

**Reply by EtherGhost** [reply]
Thanks
